addons/api/tests/unit/models/target-test.js

```diff
@@ -14,18 +14,18 @@ module('Unit | Model | target', function (hooks) {
         id: '123abc',
         type: 'target',
         attributes: {
-          host_sets: [
-            { host_set_id: '1', host_catalog_id: '2' },
-            { host_set_id: '3', host_catalog_id: '2' },
+          host_sources: [
+            { host_source_id: '1', host_catalog_id: '2' },
+            { host_source_id: '3', host_catalog_id: '2' },
           ],
         },
       },
     });
     const target = store.peekRecord('target', '123abc');
     assert.equal(
-      target.host_sets.length,
+      target.host_sources.length,
       2,
-      'Target has two entires in host_sets'
+      'Target has two entires in host_sources'
     );
     assert.equal(
       target.hostSets.length,
```

Comment:
hostSources

Reply:
This is slightly confusing I agree, but we only support a monomorphic host set relationship.  So the name must be `hostSets` until we figure out how to support truly polymorphic hostSources.